platform/feature-api/api/src/pod-api/spec.ts

```diff
@@ -108,42 +102,6 @@ export class PodSpec {
                     fc.pre(cont);
                 }
 
-                // Skipped - polyOut.writeFile is not yet used in production
-                it.skip("write/read", async () => {
-                    await fc.assert(
-                        fc.asyncProperty(pathGen, fc.fullUnicodeString(), async (path, content) => {
-                            await skipIfExists(path);
-
-                            await polyOut.writeFile(path, content, { encoding: "utf-8" });
-
-                            await assert.eventually.equal(
-                                polyOut.readFile(path, { encoding: "utf-8" }),
-                                content
-                            );
-                            await assert.eventually.deepEqual(
-                                polyOut.readFile(path),
-                                encodeUtf8(content)
-                            );
-                        })
-                    );
-                });
-
-                // Skipped - polyOut.writeFile is not yet used in production
-                it.skip("write/readDir", async () => {
```

Comment:
This one is actually activated (but simplified) in #989, so I might want to keep it to avoid conflicts.